Dockerfile

```diff
@@ -7,36 +7,38 @@ FROM alpine:3.4
 RUN apk add --update curl
 
 # add Prometheus. alas, the Prometheus developers provide no checksum
-RUN export prom=prometheus-1.3.0.linux-amd64 \
-    && curl -Lso /tmp/${prom}.tar.gz https://github.com/prometheus/prometheus/releases/download/v1.3.0/${prom}.tar.gz \
-    && tar zxf /tmp/${prom}.tar.gz -C /tmp \
+RUN export PROM_VER=1.5.2 \
+    && export prom=prometheus-${PROM_VER}.linux-amd64 \
+    && curl -Lso /tmp/prometheus.tar.gz https://github.com/prometheus/prometheus/releases/download/v${PROM_VER}/${prom}.tar.gz \
```

Comment:
Hm... should we open an issue with the Prometheus project to sign their releases?